lbz/resource.py

```diff
@@ -89,7 +90,11 @@ def __repr__(self) -> str:
         return f"<Resource {self.method} @ {self.urn} >"
 
     def _load_configuration(self) -> None:
-        self.auth_enabled = env.get("ALLOWED_PUBLIC_KEYS") or env.get("ALLOWED_AUDIENCES")
```

Comment:
Previously, this variable was set based on both `ALLOWED_PUBLIC_KEYS` and `ALLOWED_AUDIENCES`.

Reply:
I know but it was wrong.